config/types/config.go

```diff
@@ -49,36 +50,48 @@ type ConfigReference struct {
 }
 
 func init() {
-	register2_0(func(in Config, out ignTypes.Config, platform string) (ignTypes.Config, report.Report) {
-		for _, ref := range in.Ignition.Config.Append {
-			newRef, err := convertConfigReference(ref)
-			if err != nil {
-				return out, report.ReportFromError(err, report.EntryError)
+	register2_0(func(in Config, ast validate.AstNode, out ignTypes.Config, platform string) (ignTypes.Config, report.Report, validate.AstNode) {
+		r := report.Report{}
+		cfgNode, _ := getPath(ast, "ignition", "config", "append")
+		for i, ref := range in.Ignition.Config.Append {
+			tmp, _ := get(cfgNode, i)
+			newRef, convertReport := convertConfigReference(ref, tmp)
+			r.Merge(convertReport)
+			if convertReport.IsFatal() {
+				// don't add to the output if invalid
+				continue
```

Comment:
We do. It gets added to the report that gets returned. It can continue trying to parse of sections of the config and catch multiple errors at once instead of failing at the first one.